sbt_common/Makefile

```diff
@@ -3,7 +3,7 @@ include $(ROOT)/functions.Makefile
 
 STACK_ROOT 	= sbt_common
 
-SBT_DOCKER_LIBRARIES = common_elasticsearch common_messaging common_monitoring common_storage
+SBT_DOCKER_LIBRARIES = common common_elasticsearch common_messaging common_monitoring common_storage
```

Comment:
Isn’t this a NO_DOCKER library? We don’t run Docker for any common tests (we barely run any common tests, period!).